client/src/components/screen/SignIn.tsx

```diff
@@ -330,7 +344,7 @@ function SignIn(props: Props): ReactElement {
     <Container>
       <StatusBar />
 
-      <ScrollView style={{ alignSelf: 'stretch' }}>
+      <StyledScrollView media={media}>
```

Comment:
Everything looks fabulous except that we should put this `media` props here every time!